share/wake/lib/system/remote_cache_api.wake

```diff
@@ -462,10 +462,16 @@ export def rscApiGetFileBlob ((CacheSearchBlob _ uri): CacheSearchBlob) (path: S
 
     def fixupScript =
         """
-        # 1. Hardlink the file to not use 2x disk space
-        # 2. Do all edits to a tmp file then atomically mv so interruptions don't effect the build
+        # 1. No failures are acceptable
+        set -e
+        # 2. Hardlink the file to not use 2x disk space
         cp -l %{downloadPath.getPathName} '%{path}.rsctmp'
```

Comment:
Can this command fail if the `rsctmp` file already exists?

Reply:
ahh good question. I'll run a test

Reply:
interesting. It fails on a different file but seems happy if the two files are hardlinked to the same inode

Reply:
Fixed